paket.dependencies

```diff
@@ -3,6 +3,7 @@ source https://nuget.org/api/v2
 nuget FSharp.Core  redirects:force, content:none
 nuget Fable.Core
 nuget Fable.PowerPack
+nuget SourceLink.Create.CommandLine copy_local: true
```

Comment:
Hm, should this be in the Build group instead?

Reply:
Is there an issue with it there? The problem with the `Build` group was that it is limited to `framework: net46`.